src/components/Papers/PaperGroup.jsx

```diff
@@ -112,4 +97,8 @@ const PaperGroup = () => {
   )
 }
 
-export default PaperGroup
+PaperGroup.propTypes = {
+  allPapersByCategories: PropTypes.arrayOf(PropTypes.object)
+}
+
+export default memo(PaperGroup, hasPropsDeepEqual)
```

Comment:
ça ne marche pas sans `hasPropsDeepEqual` ?

Reply:
It is expensive to make the comparaison. Specially if you have a deep object. It seems that you only check the first level of the object... You should rename the method to reflect that

Reply:
> ça ne marche pas sans `hasPropsDeepEqual` ?

Pour citer la doc : `By default it will only shallowly compare complex objects in the props object.`
Ici, je voulais empêcher un second rendu "inutile", mais comparer en profondeur peu parfois s'avérer assez couteux.
Comme l'a signalé @Crash-- plus bas, il vaut sans doute mieux laisser faire...

Je vais supprimer cette fonction, ainsi que le memo qui ici ne sert à rien.